README.adoc

```diff
@@ -197,6 +197,18 @@ $ mvn clean install -Pdistribution,sample
 Note that none of those currently ships with the UI. Follow the instructions on
 link:https://github.com/hawkular/hawkular-ui-components[hawkular-ui-components] in order to run the UI.
 
+== Releasing
+
+To perform a release of Hawkular Accounts, you'll need push
+permissions to the GitHub repository and execute the following:
+
+[source,bash]
+----
+mvn release:prepare release:perform -Psample,distribution
```

Comment:
Sure, if we can come to a common ground that would be indeed great. For accounts, we can either make the "sample,distribution" part of the set of profiles that are activated by default, or we could make use of a special release profile ("-Prelease"). As it's very simple on Accounts' side, we are quite flexible here.